sk0-back/dockerfile

```diff
@@ -0,0 +1,26 @@
+# Stage 1: Build for local development
+FROM node:20-alpine as development
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
+
+RUN npm run build
+
+# Stage 2: Run
+FROM node:20-alpine
+
+ENV PORT=3000
```

Comment:
Where checks the app the environment variable? I can't see in the main.ts file it seems hard-coded:
```typescript
import { NestFactory } from '@nestjs/core';
import { AppModule } from './app.module';

async function bootstrap() {
  const app = await NestFactory.create(AppModule);
  await app.listen(3000);
}
bootstrap();```